### PR TITLE
Generate scenarios summary

### PR DESCRIPTION
**Pull request by timotheeguerin**
Summary https://github.com/Azure/cadl-ranch/blob/generate-scenarios-summary/packages/cadl-ranch-specs/cadl-ranch-summary.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +66,17 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+### Before making a Pull request
+
+Make sure to run the following commands:
+
+- `pnpm validate-scenarios`
```
Comment (iscai-msft) [inline README.md:73]: should we add this directly to the template?

File: packages/cadl-ranch-specs/cadl-ranch-summary.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
+# Cadl Ranch Project summary
+
+### MultiInterfaceClient_dogs_getDogs
+
+Illustrate grouping operations on subclient.
```
Comment (iscai-msft) [inline packages/cadl-ranch-specs/cadl-ranch-summary.md:5]: Would it be possible to parse and split this into

```md
## MultiInterfaceClient
...
### dogs_getDogs
```

Not sure if this is better, but just floating the idea

**Review by iscai-msft — COMMENTED** [review]
looks good, left some discussion comments

File: README.md (hunk 1 above)
Comment (timotheeguerin) [reply README.md:73]: well the CI will complain if you don't do it, so feels like an extra noise in the template

File: packages/cadl-ranch-specs/cadl-ranch-summary.md (hunk 2 above)
Comment (timotheeguerin) [reply packages/cadl-ranch-specs/cadl-ranch-summary.md:5]: Did think about this but I think markdown is not the best at showing the hierarchy.

Comment (timotheeguerin) [reply packages/cadl-ranch-specs/cadl-ranch-summary.md:5]: maybe what could look better is if of the title have a link to where the scenario is defined.

Comment (iscai-msft) [reply packages/cadl-ranch-specs/cadl-ranch-summary.md:5]: oh yeah that would be really cool

File: README.md [hunk 3]
```diff
@@ -60,6 +66,17 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
...
+- `pnpm validate-mockapis`
```
Comment (iscai-msft) [inline README.md:74]: I think this is `pnpm validate-mock-apis`

**Review by iscai-msft — APPROVED** [review]

**Pull request merged**